src/components/TaxonomyTerms/TaxonomyTerms.module.scss

```
@@ -0,0 +1,25 @@
+@import "../../styles/Variables";
```

We could use the base import path here to avoid the relative path traversing:

```suggestion
@import "styles/Variables";
```